x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -47,7 +50,12 @@ export { Plugin };
 export * from './plugin';
 // TODO remove this import when we expose the Rules tables as a component
 export { loadRules } from './application/lib/rule_api/rules';
+export { deleteRules } from './application/lib/rule_api/delete';
```

Comment:
I want to check something about this import to avoid our bundle getting bigger

Reply:
@XavierM Do you have any other suggestion how I could do it differently? I thought creating similar functions under observability that call directly the rules api, but I wanted to avoid duplication as much as possible. In 8.3 plan is to get rid of these imports anyway, right? Let me know what you think.

Reply:
it is not a blocker, it is more a mental note for me